Blog: v0.8 release (#896)
grouparoo/www.grouparoo.com: pages/blog/v08-release.mdx

---
title: "Grouparoo v0.8 release"
date: "2022-01-31"
author: "Brian Leonard"
pullQuote: "Grouparoo's v0.8 beings a new Model UI and other improvements."
image: "v08-release/video.jpg"
tags: [product, engineering]
---

> The v0.8 release is our first major iteration on the UI for creating your data pipeline. In the v0.7 release, we added Models, which allowed data engineers to sync multiple data schemas to Destinations. This release summarizes those Models better in the UI, giving you a clearer overview of the configuration.

Reviewer: ```suggestion
The v0.8 release is our first major iteration on the user interface for creating your data pipeline. In the v0.7 release, we added Models, which allowed data engineers to sync multiple data schemas to Destinations. This release summarizes those Models better in the UI, giving you a clearer overview of the configuration, making it quicker and easier to sync your data.
```

Here are the key features of the release.

> - Model-centric UI to see on one page where the data comes from, what it looks like, and where it is going

Reviewer 2: Not major: I wonder if "Model-centric UI" is too much of an internal term or phrasing.
"Model-based UI"
"Model-focused UI"

Wondering if maybe I'm too close to it though.

- Better filtering for Schedules and Properties

> - Upgrade to Google Sheets Source to act more like SQL Sources

Reviewer: ```suggestion
- Upgraded to Google Sheets Source to act more like SQL Sources
```

- New Destinations: Klaviyo, Vero, Salesforce Accounts and Contacts, Airtable

> ## Model UI

Reviewer: ```suggestion
## Grouparoo User Interface
```

The main update is in the UI for working with Models. We have moved the navigation to have the Models in it and each Model now has an overview with:

- Model data that includes Sources and Groups
- Sample record to work with and get a quick sense of the Model schema
- Destinations where the Model is being sent

> 

Reviewer: Maybe add a screenshot below the video too?

Here is how it works.

<p className="video-responsive">
  <iframe
    width="560"
    height="315"
    src="https://www.youtube.com/embed/Pe99Py7LZO8?rel=0"
    frameBorder="0"
    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
    allowFullScreen
  ></iframe>
</p>

This outline also follows the flow of data. We hope that, by presenting it this way, you get a better feeling for the pipeline configuration.

> As a piece of this, we have also upgraded the sample Record portion of the Model overview. In the open source Config UI, we now automatically add sample Records so it is easier to visualize your schema.

Reviewer 2: ```suggestion
As a piece of this update, we have also upgraded the sample Record portion of the Model overview. In the open source Config UI, we now automatically add sample Records so it is easier to visualize your schema.
```

## Filtering Schedules and Properties

Previously, Schedules and Properties could not be filtered by `NULL` values. We have updated the UI and normalized the JSON configuration between these and Group-building.

> Now, you can add those `NULL`-ish filters as you like.

Reviewer: ```suggestion
Now, you can add those `NULL`-ish (and non-`NULL`-ish) filters as you like.
```

<Image
  centered
  alt="Schedule filter"
  src="v08-release/filter.png"
  width={533}
  height={200}
/>

See the [upgrade notes](https://www.grouparoo.com/docs/support/upgrading-grouparoo/v07-v08) for information on how to update your configurations.

## Plugins

> [Google Sheets](https://www.google.com/sheets/about/) now acts more like SQL Sources. For example, this allows it to work better in fetching single rows from the sheet, keeping your data more consistent.

Reviewer 2: ```suggestion
[Google Sheets](https://www.google.com/sheets/about/) now acts more like SQL Sources like MySQL and PostgresQL. For example, this update allows the plugin to work better in fetching single rows from the sheet, keeping your data more consistent.
```
